Update README.md (#221)
tomquirk/linkedin-api: README.md

### How Clients query Voyager

> LinkedIn use [Rest-li Protocol]([url](https://linkedin.github.io/rest.li/spec/protocol)) an internal query language/syntax where Clients (i.e. front-ends like linkedin.com) to specify what data they want (similar to the GraphQL concept). Rest-li is implemented in the java's ecosystem with specify HTTP frameworks like [Netty](https://netty.io/). Maybe a future open source project will use this stack as a microservice for some use cases.

Reviewer: ```suggestion
linkedin.com queries data using the [Rest-li Protocol](https://linkedin.github.io/rest.li/spec/protocol). Rest-li is an internal query language/syntax where clients (like linkedin.com) to specify what data they want (similar to the GraphQL concept).
```
